code/ui/ui_main.c

```diff
@@ -519,7 +541,7 @@ int Text_Width(const char *text, float scale, int iMenuFont)
 
 	UI_WideScreenMode(qtrue);
 	w = trap_R_Font_StrLenPixels(text, iFontIndex, scale) * uiInfo.screenXFactor;
-	UI_WideScreenMode(qfalse);
+	UI_WideScreenMode(uiInfo.portraitMode);
```

Comment:
Since uiInfo.portraitMode is passed as an argument, it may be as well used in UI_WidescreenMode() directly removing dependency on it here. UI_WidescreenMode(), uiInfo.screenWidth and uiInfo.screenHeight is public interface. don't extend it unless absolutely necessary.